### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by gravytrain** [Exirel/python-xmlunittest#10]
In your documentation you show as an example : 

self.assertXpathsUniqueValue(root, ('./leaf@id', ))
self.assertXpathValues(root, './leaf@active', ('on', 'off'))

if you dont do './leaf/@active' or './leaf/@id' , it gives an error that does not clearly indicate why its broken.

**Comment by Exirel** [comment]
Sorry for the delay - I wasn't taking care of this project.

So, I'm fixing that, in the documentation first, but also now the output in such case will be (with py.test):

```
E   AssertionError: Invalid XPath expression for element root: Invalid expression
E   Xpath: ./leaf@id
E   Element:
E   <root xmlns:ns="uri">
E               <leaf id="1" active="on"/>
E               <leaf id="2" active="on"/>
E               <leaf id="3" active="off"/>
E           </root>
```

That should be better.

**Comment by Exirel** [comment]
Should be fixed by b6e02524cbda18c0f5966dfa07d72168a85d4fc3 - thanks for reporting this!
